lib/MusicBrainz/Server/Entity/Editor.pm

```diff
@@ -264,6 +264,7 @@ around TO_JSON => sub {
 
     return {
         %{$self->$orig},
+        email => $self->email,
```

Comment:
Yeah, like emails, for instance ^

If we use these `TO_JSON` representations as our standard format and base /ws/3 on them, we'll need to split them into public/private formats.